Sources/Models/API/ProductAttributesReadAPI.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ProductAttributesAPIkeysJSON.swift
+//  OpenFoodFacts
+//
+//  Created by Alexander Scott Beaty on 9/29/20.
+//
+
+import Foundation
+
+// MARK: - ProductAttributes keys
+struct ProductAttributesJson {
+    static let BarcodeKey = "code"
+    static let AttributeGroupsKey = "attribute_groups"
+
+    static func attributeGroupsLang() -> String {
+        let langCode = Bundle.main.currentLocalization
+        // language code is expected to be 2 char long
+        if langCode.count == 2 {
+            return AttributeGroupsKey + "_" + langCode
+        }
+        return AttributeGroupsKey
+    }
+}
```

Comment:
@aleene This should address the possible potential for an error with the attribute groups parameter we discussed.